go.sum

```diff
@@ -799,6 +799,8 @@ go4.org/intern v0.0.0-20211027215823-ae77deb06f29/go.mod h1:cS2ma+47FKrLPdXFpr7C
 go4.org/unsafe/assume-no-moving-gc v0.0.0-20211027215541-db492cf91b37/go.mod h1:FftLjUGFEDu5k8lt0ddY+HcrH/qU/0qk+H8j9/nTl3E=
 go4.org/unsafe/assume-no-moving-gc v0.0.0-20220617031537-928513b29760 h1:FyBZqvoA/jbNzuAWLQE2kG820zMAkcilx6BMjGbL/E4=
 go4.org/unsafe/assume-no-moving-gc v0.0.0-20220617031537-928513b29760/go.mod h1:FftLjUGFEDu5k8lt0ddY+HcrH/qU/0qk+H8j9/nTl3E=
+golang.org/dl v0.0.0-20230307162516-8f1b16c06e73 h1:Xj6VCd4FkiOl04Bqf6WKkjD4+E02celu37qSMPd55uw=
+golang.org/dl v0.0.0-20230307162516-8f1b16c06e73/go.mod h1:IUMfjQLJQd4UTqG1Z90tenwKoCX93Gn3MAQJMOSBsDQ=
```

Comment:
These changes don't seem to be needed. I'm guessing they're left over from local work? If I remove these lines and run `go mod tidy`, they don't come back, so it should be fine to delete them.